src/libcore/ops.rs

```diff
@@ -892,20 +891,22 @@ pub trait Index<Index: ?Sized> {
 /// }
 /// ```
 #[lang="index_mut"]
+#[stable]
 pub trait IndexMut<Index: ?Sized> {
     type Output: ?Sized;
 
     /// The method for the indexing (`Foo[Bar]`) operation
+    #[stable]
     fn index_mut<'a>(&'a mut self, index: &Index) -> &'a mut Self::Output;
 }
 
 /// An unbounded range.
 #[derive(Copy, PartialEq, Eq)]
 #[lang="full_range"]
-#[unstable = "API still in development"]
+#[unstable = "may be renamed to RangeFull"]
```

Comment:
If we're certain about the `Range*` types below, do we continue to be uncertain enough about this that we're not willing to settle on a name?

Also, due to the way that `&foo[]` is expanded, can you make sure that this compiles?

```rust
#![deny(unstable, warnings)]
fn main() { &"foo"[]; }
```

I'm also still a little worried about this being in the prelude, so perhaps in light of that it's ok to stay unstable for now!